include/aspect/geometry_model/two_merged_boxes.h

```diff
@@ -39,6 +39,13 @@ namespace aspect
     {
       public:
 
+        /**
+           * Initialization function. This function is called once at the
+           * beginning of the program after parse_parameters is run and after
+           * the SimulatorAccess (if applicable) is initialized.
+           */
```

Comment:
Please fix the indentation of the comment. This is unfortunately not done automatically by the indent tester.